### ISSUE TITLE
0.2.1+1-dev Too many positional arguments 

### ISSUE THREAD

**Issue by dominikheeb** [jamesblasco/modal_bottom_sheet#81]
Hi
We got a build error in our CI due to the newest dev version you published.
There seems to be an issue with a Constructor which is not correctly set:

```
../../../../tools/flutter_macos_1.20.2-stable/flutter/.pub-cache/hosted/pub.dartlang.org/modal_bottom_sheet-0.2.1+1-dev/lib/src/bottom_sheet.dart:303:43: Error: Too many positional arguments: 0 allowed, but 1 found.
Try removing the extra positional arguments.
_velocityTracker = VelocityTracker(pointerKind);


../../../../tools/flutter_macos_1.20.2-stable/flutter/packages/flutter/lib/src/gestures/velocity_tracker.dart:150:7: Context: The class 'VelocityTracker' has a constructor that takes no arguments.
class VelocityTracker {
```

**Comment by jamesblasco** [comment]
Yess sorry about that, you can use version 0.2.0 until this is solved. They have made some changes in Flutter Beta and then reversed them.

**Comment by jamesblasco** [comment]
I published 0.2.1+2 that should also solve the problem
